components/02-molecules/pull-quote/_pull-quote.scss

```diff
@@ -0,0 +1,51 @@
+:root {
+  --color-gray-light: #{$gray-light};
+  --color-gray-lightest: #{$gray-lightest};
+  --color-blue-light: #{$blue-light};
+}
+
+.pull-quote {
+  --color-pull-quote-quote: var(--color-gray-lightest);
+  --color-pull-quote-attribution: var(--color-gray-lightest);
+  --color-pull-quote-accent: var(--color-blue-light);
+}
+
+.pull-quote__figure {
+  [data-pull-quote-style='bar-left'] & {
+    padding-left: $space;
+    border-left: 0.5rem solid var(--color-pull-quote-accent);
+  }
+
+  [data-pull-quote-style='bar-right'] & {
+    padding-right: $space;
+    border-right: 0.5rem solid var(--color-pull-quote-accent);
+    text-align: right;
+  }
+
+  [data-pull-quote-style='quote-left'] & {
+    position: relative;
+    padding-left: $space;
+
+    &::before {
+      content: '“';
+      position: absolute;
+      top: 0;
+      left: 0;
+      font-size: 4em;
+      line-height: 0.8;
+      color: var(--color-pull-quote-accent);
+    }
+  }
+}
+
+.pull-quote__quote {
+  color: var(--color-pull-quote-quote);
+
+  > *:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.pull-quote__attribution {
+  color: var(--color-pull-quote-attribution);
```

Comment:
I wonder if we should use a color here that will pass accessibility tests, even though this is not meant to be a final product.

Reply:
Also, for the attribution, it might be nice to align it to the blockquote itself -- but maybe that's getting to into the design?

Reply:
@mndonx Thank you!

> Since pull quotes are generally expected to be floated to the right of content at larger widths, maybe it makes sense to go ahead and do that here? (Floating the left tends to be too disruptive.)

I feel like this would be better captured in an article/page/layout context. What do you think?

---
> I wonder if we should use a color here that will pass accessibility tests, even though this is not meant to be a final product.

I agree. The default gray background should change to white or something light. For the time being I gave the `pull-quote` element a black background. 

---

> Also, for the attribution, it might be nice to align it to the blockquote itself -- but maybe that's getting to into the design?

I adjusted this, but I'd agree it may be getting into the realm of design.

Reply:
@joetower Amanda gave me similar feedback in the modal component and I started down the same path you're doing here. However, I ended up just changing the default color palette so this warning could be prevented for all components. https://github.com/emulsify-ds/compound/pull/35/files#diff-29958d730a1ccee6be27f1c536f8bda87d21b1bedf44303a9def3d9b9512b8ccR18 

That being said, I think you can probably remove the custom property definitions in this file and just rely on the `clr()` function to print the colors we need.